plugins/ssr/ssrcommon.ml

```diff
@@ -1192,9 +1192,7 @@ let rec decompose_assum env sigma orig_goal =
 
 let tclFULL_BETAIOTA = Goal.enter begin fun gl ->
   let r, _ = Redexpr.reduction_of_red_expr (Goal.env gl)
-    Genredexpr.(Lazy {
-      rBeta=true; rMatch=true; rFix=true; rCofix=true;
-      rZeta=false; rDelta=false; rConst=[]}) in
+    Genredexpr.(Lazy Redops.all_flags) in
```

Comment:
Is this change intended? Both `rZeta` and `rDelta` used to be `false`, but are now `true`.

Reply:
not intended